Elevel/Application/Features/TopicCommands/CreateTopicCommand.cs

```diff
@@ -8,36 +8,35 @@
 
 namespace Elevel.Application.Features.TopicCommands
 {
-    public class CreateTopicCommand
+    public class CreateTopicCommand : IRequest<Guid>
```

Comment:
why? I asked several times to stick with Approach 
class Blabla
{
class Request {}
class Handler{}
class Response
|